Code/RDGeneral/types.cpp

```diff
@@ -15,85 +15,90 @@
 
 namespace RDKit {
 namespace common_properties {
```

Comment:
These need to be in the correct order, I'll make a regression test to ensure that common_properties::<Name> == "<Name>"